internal/pkg/print/markdown/table/table.go

```diff
@@ -141,18 +141,32 @@ func printOutputs(buffer *bytes.Buffer, outputs []*tfconf.Output, settings *prin
 	if len(outputs) == 0 {
 		buffer.WriteString("No output.\n\n")
 		return
+	} else if settings.InjectOutputValues != "" {
```

Comment:
You have a return right before this. This should be a `if-else` simply. Separate from the other `if`

Reply:
Depending on the guy, he might have something to say about this code, we'll see